lib/iris/coords.py

```diff
@@ -909,14 +917,25 @@ def is_compatible(self, other, ignore=None):
         return compatible
 
     @property
-    def dtype(self):
+    def points_dtype(self):
```

Comment:
Not really, no... Do you have a good reason for wanting to maintain backward compatibility here?